appinventor/buildserver/build.xml

```diff
@@ -324,6 +324,7 @@
        ===================================================================== -->
   <target name="RunLocalBuildServer"
           depends="BuildServer">
+    <property name="buildserverPassword" value="" />
```

Comment:
For running the local build I don't feel like we really should have a password, even a blank password.

Reply:
Not really, but I wanted to allow passing this parameter from CLI. In the code, if it's empty, it will treat it as if it's password disabled: https://github.com/mit-cml/appinventor-sources/pull/3471/files#diff-89c13d045c46734b3a5ec2016befa1181afe209f28df4f7cb58bbe9751a9edcaR947

This way, we can do `ant RunLocalBuildServer -DbuildserverPassword` and change it without running the JAR from Java directly.